_apprenticeship/interview-process.md

```diff
@@ -0,0 +1,36 @@
+---
+position: 42
+layout: features3col
+title: The Interview Process
+subtitle: In the process we want to learn if our expectations match, yours and ours.
+          We want to learn about your motivation and align on our expectations for the apprenticeship.
+          During the interview process we want to learn what you want to get our of the apprenticeship
```

Comment:
`to get our of the apprenticeship`  ---> `to get out of the apprenticeship`

Reply:
👍